web-client/src/modules/timeline/components/BottomPanel/BottomPanel.tsx

```diff
@@ -192,34 +196,32 @@ const ButtonContainer = styled(FlexDiv)`
     width: 100%;
   }
 `;
+/* TODO:  Check if there is any useful code here
```

Comment:
I left this code in the file to review later.  It looks like it should do something interesting but is not currently working